### PR TITLE
Subscriber test

### PR DESCRIPTION
**Pull request by azizullahsaeidi**
1. Create SubscriberFactory
2. Create SubscriberTest
3. Fixed a Farsi spell
4. Add subscriber relationship on User model

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -268,4 +268,10 @@ public function isLibraryManager($userId): ?\stdClass
             ->where('user_roles.role_id', 3)
             ->first();
     }
+
+    // Get user subscriber
+    public function subscriber(): HasOne 
+    {
+        return $this->hasOne(Subscriber::class);
+    }
```
Comment (uyscuti-wiki) [inline app/Models/User.php:276]: Duplicate. Check line 75.

File: routes/web.php [hunk 2]
```diff
@@ -257,10 +257,12 @@
...
-    Route::prefix('subscribe')->controller(SubscribeController::class)->group(function(){
-        Route::get('/', 'index')->middleware('auth')->middleware('verified');
-        Route::post('', 'store')->middleware('auth')->middleware('verified');;
-    });
+    Route::middleware('auth')->group(function(){
+        Route::prefix('subscribe')->controller(SubscribeController::class)->group(function(){
+            Route::get('/', 'index')->name('subscribe.index');
+            Route::post('', 'store')->name('subscribe.store');
+        });
+});
```
Comment (uyscuti-wiki) [inline routes/web.php:265]: Why have we removed the `verified` middleware from this route?

File: tests/Feature/Subscribers/SubscriberTest.php [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    /** @test */
+    public function en_authenticated_user_can_visit_subscribe_page(): void
+    {
+        $this->refreshApplicationWithLocale('en');
+
+        $user = User::factory()->create();
+        $response = $this->actingAs($user)->get('/en/subscribe');
+
+        $response->assertStatus(200)->assertViewIs('subscribe.index');
+    }
```
Comment (uyscuti-wiki) [inline tests/Feature/Subscribers/SubscriberTest.php:23]: 'Authenticated' and 'verified'.

**Review by uyscuti-wiki — CHANGES_REQUESTED** [review]
Hi Aziz, thank you for your work on this. I am seeing some inconsistencies with how we allow someone to subscribe. More as comments.

File: routes/web.php (hunk 2 above)
Comment (azizullahsaeidi) [reply routes/web.php:265]: We haven't removed it, `verified` was added by previous PR. when I pulled the recent changes I got `verified`

File: tests/Feature/Subscribers/SubscriberTest.php (hunk 3 above)
Comment (azizullahsaeidi) [reply tests/Feature/Subscribers/SubscriberTest.php:23]: agree with you. Thank you

File: app/Models/User.php (hunk 1 above)
Comment (azizullahsaeidi) [reply app/Models/User.php:276]: Removed. Thank you

**Comment by azizullahsaeidi** [comment]
Dear @uyscuti-wiki please review it again. Thank you

File: tests/Feature/Subscribers/SubscriberTest.php [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+        $response->assertSessionHasErrors(['email' => 'ایمیل آدرس قبلا انتخاب شده است.']);
+    }
+
+    protected function data($merge = [])
+    {
+        return array_merge(
+            [
+                'name' => 'Azizullah Saeidi',
+                'email' => 'azizullahsaeidi@email.com',
```
Comment (uyscuti-wiki) [inline tests/Feature/Subscribers/SubscriberTest.php:262]: Are you certain you want your personal info publicly available?

**Review by uyscuti-wiki — APPROVED** [review]
Looks good to me.

File: tests/Feature/Subscribers/SubscriberTest.php (hunk 4 above)
Comment (azizullahsaeidi) [reply tests/Feature/Subscribers/SubscriberTest.php]: Thank you @uyscuti-wiki , I am changing.

**Pull request merged**